pkg/loadbalancer/loadbalancer.go

```diff
@@ -475,17 +475,21 @@ func (a *L3n4Addr) DeepCopy() *L3n4Addr {
 	}
 }
 
-// Hash calculates L3n4Addr's internal SHA256Sum.
+// Hash calculates a unique string of the L3n4Addr e.g for use as a key in maps
 func (a L3n4Addr) Hash() string {
+	// Note: 32 bytes is not enough for long IPv6 addresses, but it is cheaper
+	// to reallocate on overflow than to check if a.IP is IPv6
+	b := make([]byte, 0, 32)
```

Comment:
FWIW Looks like we only need to raise to `36` bytes to spare an extra allocation in `BenchmarkL3n4Addr_Hash_IPv6_Long`.

Reply:
Yes. I'll a benchmark for the worst-case as well.